make/helpers.sh

```diff
@@ -0,0 +1,141 @@
+#!/usr/bin/env bash
```

Comment:
need to include variable initialization and validation, the rest of functions to refer as it is already technically done... just general comment not waste paper, but to keep it as a reminder.